stratum/hal/bin/bcm/standalone/start-stratum.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -ex
+
+rmmod linux_ngknet && rmmod linux_ngbde || true
+insmod linux_ngbde.ko && insmod linux_ngknet.ko
+sleep 1
+
+./stratum_bcm \
+    -external_stratum_urls=0.0.0.0:28000 \
+    -persistent_config_dir=/tmp/stratum \
+    -base_bcm_chassis_map_file=/stratum_configs/chassis_map.pb.txt \
+    -chassis_config_file=/stratum_configs/accton7710_bcm_chassis_config_onos_demo.pb.txt \
+    -bcm_sdk_config_file=/stratum_configs/AS7712-onos-demo.config.yml \
+    -forwarding_pipeline_configs_file=/tmp/stratum/pipeline_cfg.pb.txt \
```

Comment:
Fix paths to use configs from `stratum/hal/config`, i.e. `/statum_configs/<platform>/<file>`